samples/basic/servo_motor/boards/rddrone_fmuk66.overlay

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/ {
+	servo: servo {
+		compatible = "pwm-servo";
+		pwms = <&ftm0 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>, /* FMU_CH1 */
+		       <&ftm0 3 PWM_MSEC(20) PWM_POLARITY_NORMAL>, /* FMU_CH2 */
+		       <&ftm0 4 PWM_MSEC(20) PWM_POLARITY_NORMAL>, /* FMU_CH3 */
+		       <&ftm0 5 PWM_MSEC(20) PWM_POLARITY_NORMAL>, /* FMU_CH4 */
+		       <&ftm3 6 PWM_MSEC(20) PWM_POLARITY_NORMAL>, /* FMU_CH5 */
+		       <&ftm3 7 PWM_MSEC(20) PWM_POLARITY_NORMAL>; /* FMU_CH6 */
```

Comment:
note: sample will just pick 1st entry, all others can't be tested with the sample as is. In fact, a servo can only be driven by a single PWM channel. You may comment other entries, but leave them in case people wants to try.

Reply:
Done.. please check